tests/publiccloud/prepare_tools.pm

```diff
@@ -123,6 +123,16 @@ EOT
     create_script_file('terraform', '/usr/bin/terraform', $terraform_wrapper);
     record_info('Terraform', script_output('terraform -version'));
 
+    # Kubectl in a container
+    my $kubectl_version = 'latest';
```

Comment:
isnt it better to use a specific version. there are a lot of post online. i attach one random https://vsupalov.com/docker-latest-tag/#:~:text=You%20should%20avoid%20using%20the,apart%20from%20the%20image%20ID.

Reply:
Ok, Changing it

Reply:
Done